hkmc2/shared/src/test/mlscript-compile/RuntimeJS.mjs

```diff
@@ -1,9 +1,11 @@
+import Predef from "./Predef.mjs";
 
 const RuntimeJS = {
   try_catch(computation, onError) {
     try { return computation() }
     catch (error) { return onError(error) }
-  }
+  },
+  Predef
```

Comment:
This is a bit funny. Why re-export Predef here?

Reply:
This is the workaround for the cyclic import problem discussed in the meeting

Reply:
Right, and IIRC, in the meeting I explained that the proper way of solving this is to move the rendering logic to a `Rendering` module that is imported both by `Predef` and `Runtime` (where `Predef` re-exports some of its contents), not to create some cursed cyclic import.

Reply:
Moved it to a new file. I also moved some other function depended by render function (map, fold, pass, etc)